pbj-integration-tests/build.gradle.kts

```diff
@@ -57,8 +57,9 @@ testing {
     @Suppress("UnstableApiUsage")
     suites.getByName<JvmTestSuite>("test") {
         useJUnitJupiter()
-        targets.configureEach {
+        targets.all {
             testTask {
+                useJUnitPlatform()
```

Comment:
```suggestion
```

This is already set by `useJUnitJupiter()` above. See: https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html#differences_between_similar_methods_on_jvmtestsuite_and_test_task_types

Reply:
Will remove.

Reply:
Addressed in latest commit.